### ISSUE TITLE
Get media fails if Bubble Notification enabled in Chrome Beta 37

### ISSUE THREAD

**Issue by warrenjmcdonald** [rtc-io/rtc-media#15]
I have done a comparison using non SSL and a regular call to getusermedia works but if media is initialised through rtc-media the page reports unable get media. No error is raised in the JS console

Turning notification back to default allows media to be captured again.

**Comment by DamonOehlman** [comment]
@warrenjmcdonald Is this in conjunction with the [`rtc-captureconfig`](https://github.com/rtc-io/rtc-captureconfig) module?  I made some changes to package yesterday that to move generated constraints to the `optional` section (rather than `mandatory`) which was causing failures - @silviapfeiffer picked this up in one of our apps...

**Comment by warrenjmcdonald** [comment]
It is using captureconfig which is/was also causing grief on mobiles if we
set min res too high. So optional would be better.

Not sure if this is connected with the bubble notification issue unless
they have changed underlying rules.

We can get this module updated to test when doing next bundle.

**Comment by silviapfeiffer** [comment]
What is this bubble notification thing you're talking about @warrenjmcdonald ? Is that the new getUserMedia access request bubble as in chrome://flags/#enable-permissions-bubbles ? If so, that may well be a chrome bug. Did you test on apprtc, too?

**Comment by silviapfeiffer** [comment]
Incidentally, I just tried turning the bubbles on and I got an error when first loading it that way. It worked on reload though.

**Comment by warrenjmcdonald** [comment]
@silviapfeiffer yes it is the new GUM notification. I have tried with other code and do not get the error. We only getting  the issue when using the rtc-media capture ,method. Other libs like SIP.js and older VCC pages using getUserMedia npm package ( https://www.npmjs.org/package/getusermedia ) don't have the problem, 
It could be browser bug or method compatibility issue. Not a blocker, just really a heads up to report potential problem if this is turned on for testing

**Comment by silviapfeiffer** [comment]
@warrenjmcdonald that is indeed odd. Did it persist or go away after a reload, like it did for me?

**Comment by DamonOehlman** [comment]
I'll definitely have a look at this at some stage, thanks for reporting it Warren!
